src/libraries/System.Security.Cryptography/tests/AesGcmTests.cs

```diff
@@ -1005,14 +987,10 @@ public static void CheckIsSupported()
         {
             bool expectedIsSupported = !PlatformDetection.IsBrowser;
 
-            if (PlatformDetection.IsOSX)
+            if (PlatformDetection.IsOSX || PlatformDetection.UsesMobileAppleCrypto)
```

Comment:
Looking at the body of this function now, this if can just be deleted.  Neither macOS nor iOS/tvOS are browser, so it's already true, and it's not setting up an exclusion filter (in case IsOSX also implied UsesMobileAppleCrypto, which seems weird, but might be true).  So now it's

```C#
bool expect = !Browser;

if (ASpecificNotBrowser || ADifferentNotBrowser)
{
    expect = true;
}
```

which reduces to just `bool expect = !IsBrowser;`